pkg/actions/lua/lakefs/catalogexport/table_extractor.lua

```diff
@@ -0,0 +1,52 @@
+local pathlib = require("path")
+local strings = require("strings")
+local yaml = require("encoding/yaml")
+local common = require("lakefs/catalogexport/common")
+local hive = require("lakefs/catalogexport/hive")
+
+local LAKEFS_TABLES_BASE = "_lakefs_tables/"
+
+function _is_table_obj(entry, tables_base)
+    if entry == nil or entry.path_type ~= "object" then
+        return false
+    end
```

Comment:
Don't think we need to check for api bugs in this function

Reply:
you're right, removed the `nil` check, kept the object since an entry can be other things as well.